news.html

```diff
@@ -2,13 +2,11 @@
 <html lang="en">
 
 
-<head> 
+<head>
     <meta charset="utf-8">
     <title>News Page</title>
     <script src="https://use.fonticons.com/e8e95619.js"></script>
-    <link rel="stylesheet" href="main.css?002"></link>
-    <script src="https://code.jquery.com/jquery-1.11.3.js"></script>
-    <script src="new-menu.js"></script>
+    <link rel="stylesheet" href="styles/main.css?002"></link>
```

Comment:
because we put `main.css` into the `styles` folder, the path for stylesheets had to get updated.